package.json

```diff
@@ -65,13 +66,17 @@
     "test:integration:slnFilterWithCsproj": "tsc -p ./ && gulp test:integration:slnFilterWithCsproj",
     "test:artifacts": "tsc -p ./ && mocha out/test/artifactTests/**/*.test.js",
     "unpackage:vsix": "gulp vsix:release:unpackage",
-    "updatePackageDependencies": "gulp updatePackageDependencies"
+    "updatePackageDependencies": "gulp updatePackageDependencies",
+    "l10nDevGenerateLocalizationBundle": "npx @vscode/l10n-dev export --outDir ./l10n ./src",
```

Comment:
These three tasks would
1. Generate bundle.l10n.json from code.
2. Export `bundle.l10n.json` and `package.nls.json` to `vscode-csharp.xlf`
3. When `vscode-csharp.xlf` get translated, import them back to `bundle.l10.<lang>.json` and `package.nls.<lang>.json`

Reply:
So when I add a new localizable string - do I need to manually add it to a json file somewhere?  Or is that taken care of by 1)?  If so I think we should run 1) as part of the regular build step

Reply:
yeah sounds like 1 need to be added to the compile step

Reply:
Done, added to compile and compileDev